src/Raven.Server/Web/System/DatabasesHandler.cs

```diff
@@ -24,6 +24,11 @@ public class DatabasesHandler : RequestHandler
         [RavenAction("/databases", "GET")]
         public Task Databases()
         {
+            // if Studio requested information about single resource - handle it
```

Comment:
I think it is consistent with other endpoints (to have optional parameter will acts like a filter). The only change that I think we should make it to update parameter info -> name, since it is misleading.